content/community/news/science-updates/2022-12-19-nolasco-blog-post.md

```diff
@@ -0,0 +1,22 @@
+---
+date: "2022-12-19T00:00:00+00:00"
+title: 'Open Science'
+tags: ["force field", "open science", "Open Force Field", "OpenFF" ]
+categories: ["science"]
+draft: false
+description: 'By working together, we can make this happen'
+slug: 'open-science-2022-12-19'
+weight: 10
+markup: markdown
+thumb: "OpenFF-Initiative.png"
+author: "Diego Nolasco"
+---
+
+Science plays a crucial role in shaping our future and increasing knowledge is key to this progress. This is why initiatives like the Open Force Field, which aim to turn cutting-edge scientific research into valuable technology, need ongoing support. By providing long-term support to these initiatives, we can ensure that they can continue to develop and deliver comprehensive solutions that are accessible to all. It is important to recognize that the success of initiatives like the Open Force Field relies on ongoing interest and support.
+
+The concept of quality can vary depending on the context in which it is applied. In science, quality is often linked to our ability to continually add value as our understanding and capabilities evolve. In order to maintain and improve the quality of force fields, sustained investment is necessary. This is especially important if we want these force fields to keep up with the advancements in science and meet our changing needs. It is important to continue investing in initiatives like the Open Force Field, as the development of new science takes time and today's efforts can lead to progress in the future. By investing in the Open Force Field, we can ensure that we can continue to benefit from the work that has been started.
```

Comment:
"with the advancements" -> "with advancements", probably. Both are grammatically OK but I think the second reads a bit better/is closer to what we mean (because we're referring to advancements generally, not a specific set of advancements).